src/features/surveys/components/SurveySubmissionsList.tsx

```diff
@@ -85,19 +98,49 @@ const SurveySubmissionsList = ({
     },
   ];
 
+  const handlePane: GridEventListener<'cellClick'> = (params) => {
```

Comment:
I have some issues with this being a separate function so far from where the `onCellClick` property is set.

* When I read the `onCellClick` property on line 135, I was surprised to see that it handled all cells equally, even though I know that at least one of the cells should be handled separately. Once I navigated to this function, I realized that it contains the logic to exclude `respondent`.
* The name of the function is not obvious to me.
* The function seems to return something, but I don't understand what or why. Is that necessary?

I would just move the logic from this function to the `onCellClick` property directly, OR I would rename the function `handleCellClick` and reference it directly in `onCellClick={handleCellClick}`.